modules/api_loader.lua

```diff
@@ -11,6 +11,19 @@ local function get_name(xml_element)
 end
  
 
+local function dump(o)
```

Comment:
@VVeremjova as I remember we have couples of such methods in ATF. DO we rely need new one? I prefer to remove all and left just one.